sdk/tests/resources/tex-input-validation.js

```diff
@@ -260,12 +295,6 @@ var testCases = [
     expectedError: gl.INVALID_ENUM },
   { target: gl.TEXTURE_2D,
     internalFormat: gl.RGBA,
-    border: 1,
```

Comment:
Since `border` must be value 0 in GL, it is also wrong here so remove this case.